rotkehlchen/chain/ethereum/names.py

```diff
@@ -134,21 +141,23 @@ def get_prioritized_names(
 def blockchain_address_to_name(
         db: DBHandler,
         address: ChecksumEvmAddress,
+        blockchain: SupportedBlockchain,
 ) -> Optional[str]:
     """Returns the label of an ethereum blockchain account with the given address or
     None if there is no such account or the account has no label set.
     """
     with db.conn.read_ctx() as cursor:
         return db.get_blockchain_account_label(
             cursor=cursor,
-            blockchain=SupportedBlockchain.ETHEREUM,
             address=address,
+            blockchain=blockchain,
```

Comment:
why changing the order of the arguments? It doesn't matter that much but it reads better if you use the same order